client/index.html

```diff
@@ -0,0 +1,12 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Grandmother's Recipes</title>
```

Comment:
Nice!
Did we all agree on the name of our app? From what I remember, I think we hadn't settled on a final name yet.